src/main/java/lotto/Match/Match.java

```diff
@@ -0,0 +1,25 @@
+package lotto.Match;
+
+public enum Match {
+    THREE(3, 5000),
+    FOUR(4, 50000),
+    FIVE(5, 1500000),
+    SIX(6, 2000000000);
```

Comment:
enum 으로 표현을 잘 해주셨네요 👍 

```suggestion
    SIX(6, 2_000_000_000);
```
숫자를 표기할 땐 언더바(_)를 사용하면 가독성이 좋아지는 것 같아요 :)
다른곳도 체크해보고, 습관적으로 해당 컨벤션을 유지해보는게 어떨까요?